core/src/test/java/io/questdb/test/griffin/CreateTableAsSelectTest.java

```diff
@@ -87,4 +132,63 @@ private void testCreatePartitionedTableAsSelectWithOrderBy(String orderByClause)
             );
         });
     }
+
+    private void createPartitionedTableAsSelectWithOrderBy(String orderByClause, int batchSize, String o3MaxLag) throws Exception {
```

Comment:
batchSize is constant expression, there is warning for it. Please use variety of values in tests